features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/TimelinePresenter.kt

```diff
@@ -135,6 +136,12 @@ class TimelinePresenter @AssistedInject constructor(
                         )
                     } else {
                         newEventState.value = NewEventState.None
+                        if (event.firstIndex == 0) {
+                            val forwardPaginationStatus = timelineController.currentPaginationStatus(Timeline.PaginationDirection.FORWARDS)
+                            if (!forwardPaginationStatus.hasMoreToLoad) {
+                                timelineController.focusOnLive()
+                            }
```

Comment:
It's weird to get data from the controller, to then ask the controller to do the action.

Just suggesting: maybe the added method `fun currentPaginationStatus` can be removed and this `if` block could be replaced by:
```kotlin
timelineController.maybeFocusOnLive(event.firstIndex)
```